Client/Ecosia/MMP/MMP.swift

```diff
@@ -24,8 +24,7 @@ struct MMP {
                                                   installReceipt: AppInfo.installReceipt,
                                                   installTime: NSDate().timeIntervalSince1970,
                                                   updateTime: NSDate().timeIntervalSince1970)
-                let env: Environment = AppConstants.BuildChannel == .release ? .production : .staging
-                try await Singular.sendSessionInfo(appDeviceInfo: appDeviceInfo, env: env)
+                try await Singular.sendSessionInfo(appDeviceInfo: appDeviceInfo, env: .current)
```

Comment:
This is actually no longer valid on the latest Core version after the Singular update, the new way doesn't require env (see `https://github.com/ecosia/ios-browser/pull/451`).

Either way might not be relevant here if it is still using the old package version, just mentioning in case it is 👍

Reply:
Yeah it is, will merge it as is and then the `env` will go away with the SKAN merge